src/services/utilServices/MailClient.ts

```diff
@@ -5,6 +5,13 @@ import { config } from "@config/config"
 import logger from "@logger/logger"
 
 const POSTMAN_API_URL = "https://api.postman.gov.sg/v1"
+const MAIL_VERIFICATION_DELAY = 60000 // 60 seconds
+
+type MailData = {
+  id: string
+  recipient: string
+  status: string
```

Comment:
nit: this should be a sum type - `UNSENT | BOUNCE | (rest of status)`

Reply:
will do 👍